docs/howto/how_to_leverage_scenario_editor.md

```diff
@@ -0,0 +1,130 @@
+# How to Leverage Scenario Editor
+
+## Introduction
+
+Simulation plays a central role in Apollo’s internal development cycle. Dreamland empowers developers and start-ups to run millions of miles of simulation daily, which dramatically accelerates the development cycle.
+
+So far Apollo simulation allowed external users to access over 200 sample scenarios which includes a diverse range of LogSim scenarios based on real world driving data and WorldSim scenarios that have been manually created by our simulation team. To learn more about Dreamland, refer to [our Dreamland Introduction Guide](https://github.com/ApolloAuto/apollo/blob/master/docs/specs/Dreamland_introduction.md)
+
+Several developers wrote in requesting that our Dreamland platform should support Scenario Creation and Editing which the Apollo team now proudly presents in Apollo 5.0!
+
+```
+Please note,
+In order to use this feature, please send an email to our BD team at idg-apollo@baidu.com using your company email as it is currently available only for select partners
```

Comment:
Now users can submit activation request on dreamland, maybe we need update howto activate account and related screenshot

Reply:
agreed. Changed accordingly to Dreamland